metrics.go

```diff
@@ -32,6 +35,12 @@ func registerMetrics() {
 		Name:      "request_duration",
 		Buckets:   prometheus.DefBuckets,
 	}, []string{"error_code"})
+
+	msgSizeHistogram = promauto.NewHistogram(prometheus.HistogramOpts{
+		Namespace: "smtprelay",
+		Name:      "message_bytes",
+		Buckets:   []float64{1 * mb, 10 * mb, 20 * mb, 30 * mb, 40 * mb, 50 * mb},
```

Comment:
I wonder if we shouldn't start with smaller buckets here. Our max at the moment is 5MB and I doubt we'd raise it much higher (at least anytime soon?).

```suggestion
		Buckets:   []float64{0.05 * mb, 0.1 * mb, 0.25 * mb, 0.5 * mb, 1 * mb, 2 * mb, 5 * mb, 10 * mb, 20 * mb},
```

Reply:
+1, let's keep more buckets under 5 MB range

Reply:
The previous max was `10MB` and now we're bumping up the max to `50MB` to match what gmail's max is (see [here](https://support.google.com/a/answer/175121?hl=en)), unless I'm doing conversion incorrectly 😕

Reply:
Ah! Sorry, me and my assumptions :upside_down_face: 
Cool, I still think we should have buckets below the 10MB range though so we can get a sense of the distribution; if 95% of our requests are below 1MB that's expected, but if they're consistently above 10MB for example we might have some funny business going on.

Reply:
Ah good point! Sounds good :) I committed the suggestion :)

Reply:
:+1: 

If the limit is now 50MB you might want to add it in as a bucket, too.